packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -106,45 +102,15 @@ export const BootstrapCommand: ICommand = {
     // NOTE: We don't probably need this anymore, is actually not being used
     await linkProjectExecutables(projects, projectGraph);
 
-    // Bootstrap process for non Bazel packages
-    /**
-     * At the end of the bootstrapping process we call all `kbn:bootstrap` scripts
-     * in the list of non Bazel projects. We do this because some projects need to be
-     * transpiled before they can be used. Ideally we shouldn't do this unless we
-     * have to, as it will slow down the bootstrapping process.
-     */
-
-    const checksums = await getAllChecksums(kbn, log, yarnLock);
-    const caches = new Map<Project, { file: BootstrapCacheFile; valid: boolean }>();
-    let cachedProjectCount = 0;
-
-    for (const project of nonBazelProjectsOnly.values()) {
-      if (project.hasScript('kbn:bootstrap') && !project.isBazelPackage()) {
-        const file = new BootstrapCacheFile(kbn, project, checksums);
-        const valid = options.cache && file.isValid();
-
-        if (valid) {
-          log.debug(`[${project.name}] cache up to date`);
-          cachedProjectCount += 1;
-        }
-
-        caches.set(project, { file, valid });
-      }
-    }
-
-    if (cachedProjectCount > 0) {
-      log.success(`${cachedProjectCount} bootstrap builds are cached`);
-    }
-
-    await parallelizeBatches(batchedNonBazelProjects, async (project) => {
-      const cache = caches.get(project);
-      if (cache && !cache.valid) {
-        log.info(`[${project.name}] running [kbn:bootstrap] script`);
-        cache.file.delete();
-        await project.runScriptStreaming('kbn:bootstrap');
-        cache.file.write();
-        log.success(`[${project.name}] bootstrap complete`);
-      }
-    });
+    // Build typescript references
+    await spawnStreaming(
+      'node',
+      ['scripts/build_ts_refs', '--ignore-type-failures'],
```

Comment:
Add `--info` log level once https://github.com/elastic/kibana/pull/107244 is merged

Reply:
[`9b8a470` (#107217)](https://github.com/elastic/kibana/pull/107217/commits/9b8a4704ac2c2d45858df34c6b2d988634ca31e5)